config/loader.go

```diff
@@ -21,22 +59,22 @@ func LoadString(tomlText string) (*Config, error) {
 	return &config, nil
 }
 
-func LoadFile(filePath string) (*Config, error) {
+func (l *ConfigLoader) LoadFile(filePath string) (*Config, error) {
 	tomlText, err := ioutil.ReadFile(filePath)
 	if err != nil {
 		return nil, err
 	}
 
-	return LoadString(string(tomlText))
+	return l.LoadString(string(tomlText))
 }
 
-func LoadCli(c *cli.Context) *Config {
+func (l *ConfigLoader) LoadCli(c *cli.Context) *Config {
```

Comment:
Looking at this again, perhaps this should be a standalone function:

```
config := LoadCli(c)
```

The function would create a loader internally using `utils.Env()`, then return `*Config` as before.